ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/DC1-LEAF2A.yml

```diff
@@ -992,8 +996,12 @@ vlan_interfaces:
     description: Tenant_D_v6_OP_Zone_1
     shutdown: false
     vrf: Tenant_D_OP_Zone
-    ip_address_virtual: 10.4.12.254/24
-    ipv6_address_virtual: 2001:db8:412::1/64
+    ip_address: 11.4.12.2/24
+    ipv6_address: 2001:db9:412::2/64
+    ip_virtual_router_addresses:
+    - 11.4.12.1/24
```

Comment:
There should _not_ be a mask on VARP address if inside the same network as IP address. I am not sure how EOS reacts, but if it is considered "invalid" in EOS, we could add a test and capture it here.